OpenProblemLibrary/UBC/MECH/MECH2/Dynamics/Mech2_2016-2017/UBC-DYN-17-028.pg

```diff
@@ -27,6 +27,8 @@ loadMacros(
    "contextFraction.pl",
 );
 
+TEXT(beginproblem());
```

Comment:
Hi @drdrew42 , I am a co-op student at UBC and we found an error in this problem that is currently being used in our homework set. If you could approve this change so that we can give it to students right away that would be great! Thank you.

Reply:
Sure thing. You've been consistently adding to this PR, so I've let it continue. Just open a new PR with your next round of changes... 👍